Tools/autotest/sim_vehicle.py

```diff
@@ -566,6 +566,9 @@ def start_vehicle(binary, autotest, opts, stuff, loc):
     path = None
     if "default_params_filename" in stuff:
         paths = stuff["default_params_filename"]
+        if opts.uavcan:
+            suffix = paths.find(".")
+            paths = paths[:suffix]+"-uavcan"+paths[suffix:]
```

Comment:
paths = paths[:suffix] + "-uavcan" + paths[suffix:]

Reply:
see my previos comment, it will be simplier to load only the config for uavacan instead of all params

Reply:
@khancyr, I'm sorry I don't understand what you mean :(